### PR TITLE
add another user

### PR DESCRIPTION
**Pull request by dyominov**

### CODE REVIEW

File: src/main/java/com/Kh033Java/travelbook/entity/Map.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.Kh033Java.travelbook.entity;
+
+public class Map {
+}
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/entity/Map.java]: what does this class should do?

Comment (dyominov) [reply src/main/java/com/Kh033Java/travelbook/entity/Map.java]: rewrite class

File: src/main/java/com/Kh033Java/travelbook/controller/CountryController.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping("/api/v1/country")
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/controller/CountryController.java]: delete `/api/v1/`

File: src/main/java/com/Kh033Java/travelbook/dto/Map.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package com.Kh033Java.travelbook.dto;
+
+import com.Kh033Java.travelbook.entity.Country;
+import com.Kh033Java.travelbook.entity.Plan;
+import org.neo4j.ogm.annotation.Relationship;
+
+import java.util.HashSet;
+import java.util.Set;
+
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/dto/Map.java:9]: delete unused imports

File: src/main/java/com/Kh033Java/travelbook/controller/CountryController.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+public class CountryController {
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/controller/CountryController.java:22]: methods in this controller doesnt work (return nothing). delete incoming relationships in entities and rewrite CountryService

File: build.gradle [hunk 5]
```diff
@@ -45,6 +48,22 @@ dependencyManagement {
...
+//    reports {
+//        xml.enabled false
+//        html.enabled true
+//        html.stylesheet resources.text.fromFile('config/xsl/checkstyle-custom.xsl')
+//    }
+//}
+//
 //checkstyle {
 //    toolVersion = '8.1'
 //    checkstyleTest.enabled = false
```
Comment (YuliaDolenko) [inline build.gradle]: Please delete commented code

Comment (YuliaDolenko) [inline build.gradle]: Same, please delete commented code

File: build.gradle [hunk 6]
```diff
@@ -2,7 +2,7 @@ plugins {
     id 'org.springframework.boot' version '2.2.0.RELEASE'
     id 'io.spring.dependency-management' version '1.0.8.RELEASE'
     id 'java'
-    //id 'checkstyle'
+//    id 'checkstyle'
```
Comment (YuliaDolenko) [inline build.gradle]: Please delete commented code

File: src/main/java/com/Kh033Java/travelbook/service/CountryService.java [hunk 7]
```diff
@@ -1,16 +1,30 @@
...
-public interface CountryService {
+import com.Kh033Java.travelbook.entity.Country;
+import com.Kh033Java.travelbook.repository.CountryRepository;
 
-    Set<Note> getNotesByCountryByUser(String countryName, String login);
+@Service
+public class CountryService {
+	
+    @Autowired
+    CountryRepository countryRepository;
```
Comment (YuliaDolenko) [inline src/main/java/com/Kh033Java/travelbook/service/CountryService.java]: Сhange the injection annotation to the constructor and specify an access modifier

Comment (dyominov) [reply src/main/java/com/Kh033Java/travelbook/service/CountryService.java]: fix

File: build.gradle (hunk 6 above)
Comment (dyominov) [reply build.gradle]: fix

File: build.gradle (hunk 5 above)
Comment (dyominov) [reply build.gradle]: fix

**Pull request merged**